lib/standards.h

```diff
@@ -71,6 +71,17 @@ struct Standards {
         }
         return "";
     }
+    static cstd_t getC(const std::string &std) {
+        if (std == "c89") {
+            return Standards::C89;
+        } else if (std == "c99") {
```

Comment:
Nit; I'd rather not write these redundant `else`.

Reply:
So does clang-tidy ...

Reply:
Done